cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -233,7 +235,7 @@ mkTracers traceOptions tracer = do
                                                        , tot0
                                                        )
             logValue1, logValue2 :: LOContent a
-            logValue1 = LogValue "txsInMempool" $ PureI $ fromIntegral tot
+            logValue1 = LogValue "txsInMempool" $ PureI $ fromIntegral (msNumTxs tot)
```

Comment:
@CodiePP `msNumBytes tot` gives you the total size of all the transactions in the Mempool. Maybe you'd like to trace that too?

Reply:
will be added in the next release. opening an issue...